Source/lighting.cpp

```diff
@@ -12,7 +12,7 @@
 namespace devilution {
 
 LightListStruct VisionList[MAXVISION];
-BYTE lightactive[MAXLIGHTS];
+uint8_t lightactive[MAXLIGHTS];

```

Comment:
The style guide ( https://github.com/diasurgical/devilutionX/wiki/Code-Style ) lists Uint8, maybe either update that or this.

EDIT: Seems like there's a lot of standard types being swapped in by clang-tidy, not going to comment on each one.